dp/jobqueueing/jobqueueing_db.py

```diff
@@ -0,0 +1,40 @@
+import sqlalchemy
+from sqlalchemy import MetaData
+from sqlalchemy import Table, Column, Integer, String, Enum
+from sqlalchemy import DateTime, Boolean
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base(metadata=MetaData())
+metadata = Base.metadata
+
+
+class GenerateClimosQueueEntry(Base):
+    """
+    This class represents an entry in the generate_climos queue.
+    """
+    __tablename__ = 'generate_climos_queue'
+    id = Column('generate_climos_queue_id', Integer, primary_key=True, nullable=False)
+
+    # File to process
+    input_filepath = Column(String(1024), nullable=False, index=True)
+
+    # generate_climos parameters
+    output_directory = Column(String(1024), nullable=False)
+    convert_longitude = Column(Boolean, default=True, nullable=False)
```

Comment:
Again, smart to keep track of these parameters and preserve configurability through the whole stack.